backend/db/migrations/20260717120000_add_loopback.up.sql

```
@@ -0,0 +1,90 @@
+    current_round      INTEGER NOT NULL DEFAULT 0,
+    max_rounds         INTEGER NOT NULL DEFAULT 10,
+    error_message      TEXT,
+    created_by         UUID REFERENCES users(id),
+    created_at         TIMESTAMPTZ NOT NULL DEFAULT NOW(),
+    updated_at         TIMESTAMPTZ NOT NULL DEFAULT NOW()
+);
+
+-- Only 'waiting'/'active' sessions need monitoring; a partial index keeps the
+-- monitor's idle-gate (EXISTS active session) cheap.
+CREATE INDEX IF NOT EXISTS idx_loopback_sessions_active
+    ON loopback_sessions(status) WHERE status IN ('waiting', 'active');
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)